tests/unit/badge/__snapshots__/badge.spec.js.snap

```diff
@@ -1,5 +1,14 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Badge should render content without badge when badge is not visible 1`] = `
+<div
```

Comment:
But here is the opposite thing: when we have just one attribute I don't think that we should place it to separate line.
`<div className="ui-badge">` looks good enough.